dask/tests/test_base.py

```diff
@@ -224,6 +226,88 @@ def test_normalize_base():
         assert normalize_token(i) is i
 
 
+def test_normalize_object():
```

Comment:
What do you think about something like (again I've not tested the snippet below):

```python
with dask.config.set({"tokenize.allow-random": False}):
    with pytest.raises(RuntimeError, match="..."):
        tokenize(object())
```